.github/workflows/linter.yml

```diff
@@ -0,0 +1,15 @@
+name: linter
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@1.0.0
```

Comment:
Version is not enough, it needs to be a SHA: https://www.notion.so/sentry/GitHub-Actions-Guidelines-ade10a2ff0364562bd248ae92163a524

Reply:
Right! Thank you.